src/components/Portal/Portal.js

```diff
@@ -225,6 +236,7 @@ Portal.propTypes = {
 Portal.defaultProps = {
   children: null,
   focusTrap: true,
+  focusTrapOptions: null,
```

Comment:
The default should probably be an empty object. I expect that if a consumer does not provide this prop, then this component will throw an error by trying to spread `null` here:

https://github.com/carbon-design-system/ibm-security/blob/6af2669dec0cac7fc90cf29d92c408145b2d01ea/src/components/Portal/Portal.js#L194